.dockerignore

```diff
@@ -0,0 +1,11 @@
+Dockerfile
+*.md
+thirdeye-ui/
+**/target/
+# tests are not run
+**/src/test
+examples
+.idea
+.github
+doc
+tmp
```

Comment:
Sorry updated my previous message but it seems it was missed, can you please add:

```
# distribution can be built outside of docker - it will be picked if it is available
!thirdeye-distribution/target
# dependency of thirdeye-integration-tests - cannot be ignored
!thirdeye-persistence/src/test
```

Reply:
Added